src/platforms/index.mdx

```diff
@@ -7,6 +7,9 @@ redirect_from:
   - /clients/
   - /platforms/perl/
   - /platforms/node/guides/serverless-cloud/typescript/
+  - /platf
+  - /clients/perl/
+  - /platforms/perl/enriching-error-data/scopes/
```

Comment:
This redirect isn't working for some reason... maybe there's a conflict in vercel.json?